TCMSApp/src/client/app/tests/tests-create.html

```diff
@@ -115,8 +113,10 @@ <h1 class="text-center">Create New Test Case</h1>
           <div class="clearfix">
             <div class="col-sm-3">
               <div class="form-group">
-                <label for="caseAttachment">Attachment:</label>
-                <input type="file" id="caseAttachment" name="caseAttachment">
+                <input type="file" id="caseAttachment" name="caseAttachment" tabindex="-1" style="display:none">
+                <label for="caseAttachment" style="cursor:pointer">
+                  <i class="fa fa-paperclip fa-lg"></i> Attach file
+                </label>
```

Comment:
Something like this :)

![2016-01-05 15 50 11](https://cloud.githubusercontent.com/assets/14577585/12116656/144a96e2-b3c4-11e5-8636-f848abdf13f5.png)